mandatory/3-playing-computer.js

```diff
@@ -8,7 +8,7 @@
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
   2. Remove the line that throws the error.
```

Comment:
Did you run the code in your terminal?
I comment-out/removed the console,log(b) as my answer to #2.